tests/migration/Core/V6_4/Migration1665267882RenameCountryVatTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_4;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelLifecycleManager;
+use Shopware\Core\Migration\V6_4\Migration1665267882RenameCountryVat;
+use Shopware\Tests\Migration\MigrationTestTrait;
+
+class Migration1665267882RenameCountryVatTest extends TestCase
+{
+    use MigrationTestTrait;
+
+    private const NAME_EN = 'Holy See';
+    private const NAME_DE = 'Heiliger Stuhl';
+
+    private Connection $connection;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->connection = KernelLifecycleManager::getConnection();
+        $this->prepare();
+    }
+
+    public function testCountryVaticanCityExist(): void
+    {
+        $countryId = $this->connection->fetchOne('SELECT id FROM country WHERE iso = :iso AND iso3 = :iso3', ['iso' => 'VA', 'iso3' => 'VAT']);
+
+        static::assertIsNotBool($countryId, 'Country \'Vatican City\' not found');
```

Comment:
```suggestion
        static::assertIsString($countryId, 'Country \'Vatican City\' not found');
```
Is easier to understand IMHO

Reply:
BTW what are you testing here and in the next method? You are only testing the SQL statement in the test method, but not the migration, IMHO we don't need those two tests

Reply:
You are correct. Actually, my basic idea was to test that the migration update returned early when the country does not exist. I will rewrite again.